cpp/log/strict_consistent_store.h

```diff
@@ -29,8 +29,8 @@ class StrictConsistentStore : public ConsistentStore<Logged> {
 
   util::Status SetServingSTH(const ct::SignedTreeHead& new_sth) override;
 
-  util::Status AssignSequenceNumber(const int64_t sequence_number,
-                                    EntryHandle<Logged>* entry) override;
+  util::Status UpdateSequenceMapping(
+      EntryHandle<ct::SequenceMapping>* entry) override;
```

Comment:
A bit late now, but this pull request is a rather complex one, could certainly have done without the noise! :stuck_out_tongue: